src/runner/browser-env/vite/browser-modules/index.ts

```diff
@@ -1,12 +1,14 @@
 import { remote } from "webdriverio";
 import { automationProtocolPath } from "virtual:@testplane/driver";
 import { MochaWrapper } from "./mocha/index.js";
+import { initExpect } from "./expect.js";
 
 window.__testplane__.browser = window.browser = await remote({
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     automationProtocol: automationProtocolPath as any,
     capabilities: window.__testplane__.capabilities,
 });
+window.expect = initExpect();
```

Comment:
Here I can't take this declaring to `window.__testplane__` because it is always used as a global